examples/docker-compose-counter-v3.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+
+ web:
+  image: tuna/docker-counter23
+  ports:
+    - "5000:5000"
+  deploy:
+    restart_policy:
+      condition: any
+  labels:
+    kompose.service.type: NodePort
+
+ redis:
+  image: redis:3.0
+  deploy:
+     replicas: 2
```

Comment:
@surajnarwade Yes, I know, but I don't think you understand that replicas won't work with syncing the redis databases. They won't automatically sync / connect as a cluster and thus your example won't work.

If you deployed with 2 replicas, and then changed it to 1 or somehow one of the containers was re-created on Kubernetes, you'll lose the data.

@kadel help me out here :)